foia_hub/templates/includes/nav.html

```diff
@@ -16,8 +16,19 @@
     <div class="container">
       <div id="notice--close" tabindex="0" aria-label='Select to hide banner'><i class="fa fa-times"></i></div>
       <div class="notice--content">
```

Comment:
What about adding a different background color so those who have been to the site before are more likely to notice the new text? #e31c3d is one of the main USWDS colors and might be a good one.

Reply:
Maybe a little intense... the contrast also looks a little poor.
![screenshot from 2017-09-19 14-37-53](https://user-images.githubusercontent.com/509703/30617442-e84329e8-9d4a-11e7-8bfc-51a3d4b0f2dc.png)

Reply:
Alternatively, we could disable the 'X' to close, it's just a static banner.